boards.txt

```diff
@@ -582,6 +582,19 @@ Disco.build.core=arduino
 Disco.build.board=Disco
 Disco.build.extra_flags=-D{build.product_line} {build.enable_usb} {build.xSerial}
 
+# DISCO_B_G431B_ESC1 board
+Disco.menu.pnum.DISCO_B_G431B_ESC1=B-G431B-ESC1
+Disco.menu.pnum.DISCO_B_G431B_ESC1.node="NODE_G431CB,NOD_G431CB"
```

Comment:
I had to add "DIS_G431CB" to the node line above in order to get the programmer to recognise the storage device. I was using a brand new board (no changes to the ST Link FW).